### PR TITLE
Feature/map explorer locale

### PR DESCRIPTION
**Pull request by Zerro97**
## Changes
1. Sends messages (locale code) to compose multiplatform on locale change
2. Added translation for simplified chinese & traditional chinese
3. Added placeholder for german
4. Adjusted korean translation

### CODE REVIEW

File: frontend/src/views/MapExplorerStepsView.vue [hunk 1]
```diff
@@ -1,12 +1,22 @@
...
-const MAPEXPLORER_URL = import.meta.env.VITE_MAPEXPLORER_URL || 'https://stefan-oltmann.de/oni-seed-browser';
+import { ref } from 'vue';
+
+const MAPEXPLORER_URL = import.meta.env.VITE_MAPEXPLORER_URL || 'http://localhost:8080/';
 const iframeUrl = MAPEXPLORER_URL
+const iframeRef = ref(null)
+
+function sendEvent() {
+  if (iframeRef.value && iframeRef.value.contentWindow) {
+    iframeRef.value.contentWindow.postMessage("test data", MAPEXPLORER_URL);
```
Comment (StefanOltmann) [inline frontend/src/views/MapExplorerStepsView.vue]: That looks experimental to me and not done.

**Comment by StefanOltmann** [comment]
My 2 cents: I think we don't need placeholder translations. If one shows up, it should actually work and be complete.

File: frontend/src/views/MapExplorerStepsView.vue [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
-  },
-  mounted() {
-    this.queryParams = this.$route.query;
-    this.queryParams.embedded = 'mni'; 
-
-    let url = MAPEXPLORER_URL;
-    if (this.$route.params.seed) {
-      url = `${url}#${this.$route.params.seed}`;
-    }
-    url = `${url}?${new URLSearchParams(this.queryParams).toString()}`;
```
Comment (barratt) [inline frontend/src/views/MapExplorerStepsView.vue:26]: This has been deleted but I don't see it re-implemented. By using this if we add ?lang = de, cn, en etc it should tell steps' lib the new language, it also removed the ?embedded=mni query tag too

Comment (barratt) [reply frontend/src/views/MapExplorerStepsView.vue:26]: This code essentially passes down any query params from mapsnotincluded to steps map-explorer url

File: frontend/src/views/MapExplorerStepsView.vue (hunk 1 above)
Comment (Zerro97) [reply frontend/src/views/MapExplorerStepsView.vue]: The latest commit should pass actual locale code and uses appropriate url

File: frontend/src/views/MapExplorerStepsView.vue (hunk 2 above)
Comment (Zerro97) [reply frontend/src/views/MapExplorerStepsView.vue:26]: I'll add this back in!

Comment (Zerro97) [reply frontend/src/views/MapExplorerStepsView.vue:26]: I've made the changes based on the feedback. Let me know if further adjustments are needed

**Comment by barratt** [comment]
Code looks good, test data was removed, query string was placed back, checked it ran in a codespace (we should make tests for this)

**Pull request merged**